webpack.config.js

```diff
@@ -56,7 +55,7 @@ module.exports = {
       },
       {
         test: /\.js$/,
-        exclude: /node_modules/,
+        exclude: /node_modules|webcomponents/,
```

Comment:
Why are you excluding them? 🤔

Reply:
Mmm,I did it because it threw an error and failed to instantiate the webcomponent because of an error with the constructor, but excluding them would result in incompatibility with IE11 right?